### PR TITLE
docs: leave ledger and carry forward expiry

### PR DESCRIPTION
**Pull request by Mangesh-Khairnar**
- Added docs for leave ledger entry in other section of the human resource section.

Screenshot 2019-05-29 at 12 59 09 PM

- Leave Ledger Entry

Screenshot 2019-05-29 at 12 59 18 PM

- Carry forward leave expiry in leave type

Screenshot 2019-05-29 at 12 58 55 PM

### CODE REVIEW

**Comment by Mangesh-Khairnar** [comment]
docs for https://github.com/frappe/erpnext/pull/17624

File: erpnext_com/www/docs/user/manual/en/human-resources/leave-ledger-entry.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<!-- add-breadcrumbs -->
+# Leave Ledger Entry
+
+A unified ledger for all leaves related transactions for an employee. This maintains all the leaves related transactions, this includes the leave allocations, leave applications and leave encashments related to your employees.
+
+<img alt="Leave Ledger Entry" class="screenshot" src="{{docs_base_url}}/assets/img/human-resources/leave-ledger-entry.png">
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/human-resources/leave-ledger-entry.md]: image has a black bar at the top, crop it

File: erpnext_com/www/docs/user/manual/en/human-resources/leave-ledger-entry.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<!-- add-breadcrumbs -->
+# Leave Ledger Entry
+
+A unified ledger for all leaves related transactions for an employee. This maintains all the leaves related transactions, this includes the leave allocations, leave applications and leave encashments related to your employees.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/human-resources/leave-ledger-entry.md]: change all leaves to leave in this paragraph
"encashments of your employees"

File: erpnext_com/www/docs/user/manual/en/human-resources/leave-ledger-entry.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/human-resources/leave-ledger-entry.md]: add some more content. about filtering etc.,

File: erpnext_com/www/docs/user/manual/en/human-resources/leave-type.md [hunk 4]
```diff
@@ -26,6 +26,12 @@ requirement.
 
 * Is Optional: Check this Optional Leaves are holidays which Employees can choose to avail from a list of holidays published by the company. The Holiday List for optional leaves can have any number of holidays but you can restrict the number of such leaves granted to an Employee in a Leave Period by setting the Max Days Leave Allowed field.
 
+* Carry Forward: It is possible that the carry forwarded leaves needs to expire within a particular time period based on the company policies regarding carry forwarded leaves. Once the "Is Carry Forward" is set, you can set the number of days in "Carry Forward Leave Expiry" field to define the expiry time period.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/human-resources/leave-type.md]: remove s, "need". end sentence at policies.

File: erpnext_com/www/docs/user/manual/en/human-resources/leave-type.md [hunk 5]
```diff
@@ -26,6 +26,12 @@ requirement.
...
+
+	<img class="screenshot" alt="Carry Forward Leave Expiry" src="{{docs_base_url}}/assets/img/human-resources/carry-forward-leave-expiry.png">
+
+	>Note: On setting carry forward leave expiry as 0, unused leaves will not expire at the end of the leave allocation, hence, both previous remaining leaves and carry forwarded leaves will be considered for carry forward calculation for next leave allocation.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/human-resources/leave-type.md]: new sentence from . Hence
both existing leaves and carry forwarded...

**Comment by scmmishra** [comment]
@prasadarr please recommend your changes as suggestions ([ref](https://help.github.com/en/articles/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request))

Will save both the reviewer and author a lot of time.

**Comment by prasadarr** [comment]
> @prasadarr please recommend your changes as suggestions ([ref](https://help.github.com/en/articles/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request))
> 
> Will save both the reviewer and author a lot of time.

yeah, wasn't aware of that option when this was reviewed

File: erpnext_com/www/docs/user/manual/en/human-resources/leave-ledger-entry.md (hunk 1 above)
Comment (Mangesh-Khairnar) [reply erpnext_com/www/docs/user/manual/en/human-resources/leave-ledger-entry.md]: removed

**Pull request merged**